src/main/java/by/iba/party/service/impl/UserServiceImpl.java

```diff
@@ -1,18 +1,20 @@
 package by.iba.party.service.impl;
 
-import by.iba.party.entity.Party;
+import by.iba.party.dto.UserDto;
 import by.iba.party.entity.User;
+import by.iba.party.exception.NoEntityException;
+import by.iba.party.mapper.UserMapper;
 import by.iba.party.repository.UserRepository;
 import by.iba.party.service.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
-import java.util.Optional;
 
 @Service
 public class UserServiceImpl implements UserService {
     private final UserRepository userRepository;
+    private final UserMapper userMapper = UserMapper.INSTANCE;
```

Comment:
MapStruct supports spring component model `@Mapper(componentModel="spring", uses=MyService.class)` so you can do `@Autowired` mappers. Please utilize this approach, instead of static assignment.